joern-cli/frontends/csharpsrc2cpg/src/main/scala/io/joern/csharpsrc2cpg/astcreation/AstCreatorHelper.scala

```diff
@@ -77,9 +78,19 @@ trait AstCreatorHelper(implicit withSchemaValidation: ValidationMode) { this: As
         identifierNode(dotNetNode.orNull, x.name, x.name, Defines.Any)
   }
 
+  protected def signatureToTypeString(argTypeString: String): String = {
```

Comment:
I'd prefer this functionality to be in `CSharpScope` so that you don't have to do unnecessary string manipulation